testing/tests/DevExpress.viz.gauges/barGauge.tests.js

```diff
@@ -574,6 +574,29 @@ QUnit.test('Some values are not changed', function(assert) {
     }, this);
 });
 
+QUnit.test('resolveLabelOverlapping - shift', function(assert) {
+    const done = assert.async();
+    this.$container.dxBarGauge({
+        values: [10, 11, 12],
+        resolveLabelOverlapping: 'shift'
+    });
+    const group = this.getBarsGroup();
+
+    group.animationComplete = $.proxy(function() {
+        assert.deepEqual(group.children[2]._stored_settings.points, [64.04786, 219.49845, 21, 7], 'line 1 coords');
+        assert.equal(group.children[3]._stored_settings.x.toFixed(3), 	34.027, 'text 1 is coord x');
+        assert.equal(group.children[3]._stored_settings.y.toFixed(3), 222.679, 'text 1 is coord y');
+        assert.deepEqual(group.children[6]._stored_settings.points, [95.86324, 203.76216, 12, 17], 'line 2 coords');
+        assert.equal(group.children[7]._stored_settings.x.toFixed(3), 31.813, 'text 1 is coord x');
```

Comment:
`text 1` is used on every check. There is should be `text 2`

Reply:
fixed